queries/flights.py

```diff
@@ -112,21 +122,18 @@ def help_text(lemma: str) -> str:
     | "hver" "er" "brottfarartími"
     | "hver" "er" "brottfarartíminn" "fyrir"
 
-$score(+35) QFlightsWhenDep
-
 QFlightsWhen →
     "hvenær" | "klukkan" "hvað"
 
-$score(-35) QFlightsWhen
-
 QFlightsPlane →
     "flug" | "flugvél" | "flugvélar" | "vél" | "vélar" | "flugs"
```

Comment:
If we wanted to be very precise, we would have two functions here: QFlightsPlane_nf and QFlightsPlane_ef. Even more fancy would be:

`QFlightsPlane/fall -> 'flug:hk'_et/fall | 'flugvél:kvk'_et/fall | 'vél:kvk'_et/fall`

This would catch the different grammatical forms automatically, including indefinite or definite ('vél' / 'vélin' etc.)

Reply:
That is a lot cleaner. Fixed it, thank you!